pkgs/applications/networking/cluster/k9s/default.nix

```diff
@@ -1,28 +1,45 @@
-{ stdenv, lib, buildGoModule, fetchFromGitHub }:
+{ stdenv, lib, buildGoModule, fetchFromGitHub, installShellFiles, testers, k9s }:
 
 buildGoModule rec {
   pname = "k9s";
-  version = "0.25.18";
+  version = "0.25.21";
 
   src = fetchFromGitHub {
     owner  = "derailed";
     repo   = "k9s";
     rev    = "v${version}";
-    sha256 = "sha256-iUhMPtFX7qFULegiyhlT4aG9q3deZ8aRqyEcbZ9jY/s=";
+    sha256 = "sha256-ziIMTMK6G8vXje6GWPvcIWmlubq75XVrJUzZlA+R0Rc=";
   };
 
   ldflags = [
     "-s" "-w"
     "-X github.com/derailed/k9s/cmd.version=${version}"
     "-X github.com/derailed/k9s/cmd.commit=${src.rev}"
+    "-X github.com/derailed/k9s/cmd.date=1970-01-01T00:00:00Z"
   ];
 
-  vendorSha256 = "sha256-mMob7M9RQlqaVK0DgHpaAK9d1btzfQetnliUqFTvjJQ=";
+  tags = [ "netgo" ];
 
-  preCheck = "export HOME=$(mktemp -d)";
+  vendorSha256 = "sha256-wL8Unht/ZRAGDuC/U4SFV5PdExy78F4DMyM8+7CMtOY=";
 
   # TODO investigate why some config tests are failing
   doCheck = !(stdenv.isDarwin && stdenv.isAarch64);
+  # Required to workaround install check error:
+  preCheck = "export HOME=$(mktemp -d)";
```

Comment:
Do you still need this? Doesn't seem like you're doing any kinda of install check.